src/components/ProgressIndicator/ProgressIndicator.tsx

```diff
@@ -93,9 +97,10 @@ export const ProgressIndicator: ChakraComponent<
       // Only display the percentage text for the default size, not in the
       // indeterminate state, and when `showLabel` is true.
       if (indicatorType === "circular") {
-        // For the small size, since the label won't be visible, we need to add
-        // it to the parent component's `aria-label` attribute.
-        if (size === "small") {
+        // For the small size and the bottom label placement, since the
+        // label won't be visible, we need to add it to the parent
+        // component's `aria-label` attribute.
+        if (size === "small" && labelPlacement !== "right") {
```

Comment:
Nit: I would personally make this `&& labelPlacement === "bottom"` for simpler readability.